src/GitHub.Api/ApiClientConfiguration_User.cs

```diff
@@ -1,11 +1,13 @@
+﻿using System;
+
 namespace GitHub.Api
 {
-    public partial class ApiClient : IApiClient
+    static partial class ApiClientConfiguration
```

Comment:
These IDs need to be in the code, users can't be configuring this data manually on their systems in order to get the extension working. We can gitignore the file that keeps these variables in the public build so people can configure and build with them without having the repo dirtied or accidentally pushed.